site/admission/index.html

```diff
@@ -48,18 +48,20 @@ <h5>Students must live in Boston by TKTK in order to enroll. Learn more</h5>
   <h2>Upcoming Admissions Events</h2>
   <p>View All</p>
   <div>
+    <img src="../assets/images/placeholder-image-rectangle.png" alt="placeholder-image-rectangle" width="275"
```

Comment:
You should remove the height and width properties from all the images. It's not needed.

Reply:
You can do this on all the images.